src/ServicePulse.Host/vue/src/resources/SortOptions.ts

```diff
@@ -0,0 +1,13 @@
+import FailedMessage from "./FailedMessage";
+import FailureGroup from "./FailureGroup";
+
+export type GroupType = FailedMessage | FailureGroup;
+export type GroupPropertyType = string | number | Date;
+
+export default interface SortOptions {
+  description: string;
+  dir: string;
```

Comment:
I would prefer 'direction'

Reply:
This change is better done after the other components are migrated to ensure we don't mess up any references